x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -701,7 +700,16 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
               : {}),
           };
         }),
-      [columnHeaders, defaultCellActions, browserFields, data, pageSize, id, dispatch]
+      [
+        browserFields,
+        columnHeaders,
+        data,
+        defaultCellActions,
+        disabledCellActions,
+        dispatch,
+        id,
+        pageSize,
+      ]
```

Comment:
Interesting, did you use a prettier plugin to sort hooks deps?

Reply:
prettier formatted them one per line; the sort was powered by a 🐒